### PR TITLE
feat(grounds-appeal): add grounds of appeal

### PR DESCRIPTION
**Pull request by Fenisay**
## Ticket Number

UCD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] I have merged commits to avoid fixing things in this PR
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: forms-web-app/src/views/submit-appeal/grounds-appeal.njk [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+              </li>
+            </ul>
+          </div>
+        </details>
+        <div class="govuk-inset-text">File size should be no more than 1.5GB</div>
+        <div class="govuk-form-group">
+          <label class="govuk-heading-m" for="appeal-statement">
+            Upload a file
+          </label>
+          <input value="{{ data['appeal-statement'] }}" class="govuk-file-upload" id="appeal-statement"
```
Comment (a6software) [inline forms-web-app/src/views/submit-appeal/grounds-appeal.njk:105]: please could we use the gov uk component for this?

https://design-system.service.gov.uk/components/file-upload/

File: forms-web-app/src/views/submit-appeal/grounds-appeal.njk [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            </ul>
+          </div>
+        </details>
+
+        <div class="govuk-form-group">
+          <div class="govuk-checkboxes">
+            <div class="govuk-checkboxes__item">
+              <input class="govuk-checkboxes__input" id="sense-check" name="sense-check" value="on" type="checkbox">
+              <label class="govuk-label govuk-checkboxes__label" for="sense-check">
+                I confirm that I have not included sensitive information in my appeal statement.
```
Comment (a6software) [inline forms-web-app/src/views/submit-appeal/grounds-appeal.njk:132]: Please could we use the gov uk component here?

https://design-system.service.gov.uk/components/checkboxes/

File: forms-web-app/src/views/submit-appeal/grounds-appeal.njk [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+              </label>
+            </div>
+          </div>
+        </div>
+
+
+        <div class="govuk-form-group">
+          <button class="govuk-button" data-module="govuk-button">
+            Save and continue
```
Comment (a6software) [inline forms-web-app/src/views/submit-appeal/grounds-appeal.njk:141]: Please could we use the gov uk button component here?

File: forms-web-app/src/views/submit-appeal/grounds-appeal.njk [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+          </button>
+        </div>
+        <input type="hidden" name="sense-check" value="{{ data['appeal-confirmation'] }}">
```
Comment (a6software) [inline forms-web-app/src/views/submit-appeal/grounds-appeal.njk:144]: is this `data` var working ?

File: forms-web-app/src/views/submit-appeal/grounds-appeal.njk (hunk 1 above)
Comment (Fenisay) [reply forms-web-app/src/views/submit-appeal/grounds-appeal.njk:105]: Yes at the moment, it was just a dumb copy paste from the prototype, all the logics are in other US 
https://pins-ds.atlassian.net/browse/UCD-832
for example

**Pull request closed without merging**